src/web/hello_js/hello_js.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Hello JS
+level: 1
+author: Arve Seljebu
+---
+
+# Introduksjon {.intro}
+I denne oppgaven blir du introdusert for programmeringsspråket JavaScript. Du
+skal gjøre den klassiske oppgaven *Hello world*, med en katt.
+
+![](jsbin.png)
+
+# Steg 1: Bruke JS Bin {.activity}
+Du skal bruke en nettside som heter *JS Bin* for denne oppgaven. Dette fordi
+JS Bin ser likt ut i alle nettlesere, så da er det enklere å følge stegene.
+
+## Sjekkliste {.check}
+
+- Åpne [JS Bin](http://jsbin.com/?js,console) i et eget vindu.
+- Legg merke til at du får to faner; **JavaScript** og **Console**.
+- Koden skrives i **JavaScript**.
+- Og i **Console** vises tekstbeskjeder som blir skrevet ut.
+- Skriv inn denne koden:
+
+  ```js
+  console.log("hello world");
+  ```
+
+- Trykk **Ctrl** + **Enter** (eller bruk knappen <button>Run</button>) og se hva som skjer
+  i **Console**.
+- Kan du endre koden til å skrive ditt navn?
+
+## Forklaring {.tip}
+Her er en forklaring til koden over:
+- `console.log()` betyr at vi ønsker å kjøre kommandoen med navnet `console.log`.
+- `.log` er en av flere utskriftsmetoder. Prøv `console.error`, ser du hva
+  forskjellen til `console.log` er?
+- Tekst skrives mellom `"`, slik som `"hello world"`.
+- `;` betyr at kodesetningen er ferdig.
+
+# Steg 2: Lage en funksjon {.activity}
+En funksjon er nesten som en oppskrift. Funksjoner brukes når man ønsker å
+gjenbruke kode flere ganger eller bryte opp et problem til mindre biter. En
+funksjon kan både ta i mot og returnere data. Din funksjon skal hete `hei`,
+ta i mot et `navn` og skrive navnet til konsollen.
+
+## Sjekkliste {.check}
+
+- Slett koden din fra steg 1.
+- Skriv inn denne koden:
+
+  ```js
+  function hei (navn) {
+    console.log(navn);
+  }
+  ```
+
+- `function hei` betyr at funksjonen skal hete `hei`.
+- `(navn)` betyr at vi kan sende inn data til funksjonen.
+- `{` bestemmer starten til funksjonen.
+- `}` bestemmer slutten til funksjonen.
+- Mellom `{` og `}` er det som funksjonen skal gjøre.
+- Akkurat nå gjør ikke funksjonen så mye, men la oss teste den ut. Skriv dette
+  etter funksjonen:
+
+  ```js
+  hei("Lars");
+  hei("Jens");
+  ```
+
+- `hei("Lars")` betyr at vi skal kjøre funksjonen og sende inn `"Lars"`.
+- Trykk **Ctrl** + **Enter** for å kjøre koden.
+- Ser du både `"Lars"` og `"Jens"` i konsollen?
+
+## Tips {.tip}
+Du kan slette historien i **Console** ved å trykke på knappen <button>Clear</button>.
```

Comment:
Siden det er [så mange metoder](https://developer.mozilla.org/en-US/docs/Web/API/Console), tenker jeg en holder. Synes det er fint å nevne, ettersom det kan være forvirrende for de som kommer fra C-like språk med `print()` eller lignende. Det var i alle fall min umiddelbare tanke ved første syn av `console.log`: Hvorfor `.log`?